### ISSUE TITLE
Names not easy to find

### ISSUE THREAD

**Issue by clehner** [ssbc/patchwork#289]
Is there a way to see what names have been given to a feed by one's network?

I was followed by @DTNmX+4SjsgZ7xyDh5xxmNtFqa6pWi5Qtw7cE8aR9TQ=.ed25519 which did not publish a name. I inferred from a post and the data log page that it was a pub. However, I did not find in the UI a list of names that people have given the feed. I obtained this info instead using the command line:
```
sbot links --dest @DTNmX+4SjsgZ7xyDh5xxmNtFqa6pWi5Qtw7cE8aR9TQ=.ed25519 --values --rel about | grep name
```

It would be nice to see this info in the UI somewhere on the profile page, maybe as suggestions when renaming the feed, or listed under the hex name, or in a "Links" log on the profile page

**Comment by dominictarr** [comment]
I agree. the social nature of naming needs to be more visible.
this is also relavant: https://github.com/ssbc/patchwork/pull/251

**Comment by clehner** [comment]
Fixed by #301

**Comment by pfraze** [comment]
:+1:
